docs/hosting-a-server-with-northstar/prerequisites.md

```diff
@@ -60,12 +59,4 @@ If you accidentally click the deny button, then follow the step to allow it.
 To check whether you set everything up correctly, start the game via Northstar and go into a private match. Another Northstar user should now be able to see your server on the server browser and join it.\
 You can also use a web based server browser like the one made by [Taskinoz](https://taskinoz.com/northstar/) or [cpdt](https://cpdt.dev/northstar/) to see if your server shows up in there.
 
-You can also use a third-party site like [this one](https://www.ipfingerprints.com/portscan.php) to see whether you set up the TCP port forwarding correctly for the authentication port (usually `8081`) by performing a scan while your Northstar server is running.
-
-![Example check](https://raw.githubusercontent.com/R2Northstar/NorthstarWiki/main/docs/images/portforwarding-testing.png)
-
-![Working portforwarding for the TCP port](https://raw.githubusercontent.com/R2Northstar/NorthstarWiki/main/docs/images/portforwarding-working.png)
-
-Testing port forwarding like this only works for the TCP port due to the way UDP and Northstar work.
-
-Note that by default your server is called `Unnamed Northstar Server`. You can change this name as described in the next page.
+Note that by default your server is called `Unnamed Northstar Server`. You can change this name as described in the next page.
```

Comment:
No clue why GitHub still keeps highlighting this as changed. It neither has trailing whitespace nor missing newline so no clue what's causing it

Reply:
I swear if this is `CRLF` vs `LF` I'm gonna be really annoyed >:(

Reply:
Oh apparently it was missing a newline before. Wtf?

Reply:
aaaand that matters how?

Reply:
Oh no, nevermind. This PR removes the newline. And GitHub vscode extension just ignored that in the diff, ugh. Had to go in manually by checking out branch locally to detect.

Reply:
> aaaand that matters how?

Affects the diff and `git blame` and so it should be avoided if possible. Not the end of the world but stuff like this makes things messy in the long run. Anyway I fixed it manually now.